src/tsickle.ts

```diff
@@ -920,6 +925,9 @@ class Annotator extends ClosureRewriter {
             if (sym.flags & ts.SymbolFlags.Alias) {
               sym = typeChecker.getAliasedSymbol(sym);
             }
+            if (this.newTypeTranslator(heritage).isBlackListed(sym)) {
```

Comment:
Nope, I slipped in finding the closest context node (not that it matters). Fixed.